pkg/loadbalancer/loadbalancer.go

```diff
@@ -95,8 +115,13 @@ func (s *LBSVC) GetModel() *models.Service {
 		BackendAddresses: make([]*models.BackendAddress, len(s.BES)),
 	}
 
+	placements := make(backendPlacements, len(s.BES))
 	for i, be := range s.BES {
-		spec.BackendAddresses[i] = be.GetBackendModel()
+		placements[i] = backendPlacement{pos: i, id: be.ID}
+	}
+	sort.Sort(placements)
```

Comment:
Instead of implementing the sort interface, you have the https://golang.org/pkg/sort/#Slice